builtin/mainmenu/dlg_create_world.lua

```diff
@@ -78,6 +78,17 @@ local mgv6_biomes = {
 	},
 }
 
+local mapgens_descriptions = {
+	v7 = fgettext("Default mapgen with large complex mountins and plains."),
+	valleys = fgettext("Large valleys with complex terrain and rivers."),
+	carpathian = fgettext("Realistic looking world with vast plains."),
+	v5 = fgettext("Old mapgen."),
+	flat = fgettext("World Flat terrain."),
+	fractal = fgettext("Wold with fractal structure."),
+	singlenode = fgettext("Empty world, use for lua defined mapgens."),
+	v6 = fgettext("Simple mapgen with few features, not recommended."),
+}
```

Comment:
nitpick: none of these are full sentences so there should be no full stop at the end